### ISSUE TITLE
CSV::MalformedCSVError: Illegal quoting on a line like 1,Glk,Lev "The Black Spider" Yashin,USSR

### ISSUE THREAD

**Issue by shvetsovdm** [BrewhouseTeam/csv-importer#25]
irb example:
```shell
2.1.5 :023 > str = "1,Glk,Lev \"The Black Spider\" Yashin,USSR"
 => "1,Glk,Lev \"The Black Spider\" Yashin,USSR"
2.1.5 :024 > CSV.parse(str)
CSV::MalformedCSVError: Illegal quoting in line 1.
	from /Users/shvetsovdm/.rvm/rubies/ruby-2.1.5/lib/ruby/2.1.0/csv.rb:1857:in `block (2 levels) in shift'
	from /Users/shvetsovdm/.rvm/rubies/ruby-2.1.5/lib/ruby/2.1.0/csv.rb:1819:in `each'
	from /Users/shvetsovdm/.rvm/rubies/ruby-2.1.5/lib/ruby/2.1.0/csv.rb:1819:in `block in shift'
	from /Users/shvetsovdm/.rvm/rubies/ruby-2.1.5/lib/ruby/2.1.0/csv.rb:1779:in `loop'
	from /Users/shvetsovdm/.rvm/rubies/ruby-2.1.5/lib/ruby/2.1.0/csv.rb:1779:in `shift'
	from /Users/shvetsovdm/.rvm/rubies/ruby-2.1.5/lib/ruby/2.1.0/csv.rb:1721:in `each'
	from /Users/shvetsovdm/.rvm/rubies/ruby-2.1.5/lib/ruby/2.1.0/csv.rb:1735:in `to_a'
	from /Users/shvetsovdm/.rvm/rubies/ruby-2.1.5/lib/ruby/2.1.0/csv.rb:1735:in `read'
	from /Users/shvetsovdm/.rvm/rubies/ruby-2.1.5/lib/ruby/2.1.0/csv.rb:1293:in `parse'
	from (irb):24
	from /Users/shvetsovdm/.rvm/rubies/ruby-2.1.5/bin/irb:11:in `<main>'
```

**Comment by shvetsovdm** [comment]
if we provide a `quote_char` option we will solve the issue (as suggested here http://stackoverflow.com/a/12533730)

irb example
```shell
2.1.5 :026 > str = "1,Glk,Lev \"The Black Spider\" Yashin,USSR"
 => "1,Glk,Lev \"The Black Spider\" Yashin,USSR"
2.1.5 :027 > CSV.parse(str, quote_char: "\x00")
 => [["1", "Glk", "Lev \"The Black Spider\" Yashin", "USSR"]]
```

**Comment by pcreux** [comment]
Fixed by #26. Thanks again @shvetsovdm !
